CHANGELOG.rst

```diff
@@ -5,8 +5,16 @@ ChangeLog
 master (unreleased)
 ===================
 
-- Allow POST method for form validation endpoint. This version is the last to support the GET verb for the validation endpoint. We'll drop it in the next version.
+Nothing here yet.
+
+Realease 1.3.0 (2018-02-14)
```

Comment:
:smiley:

Reply:
ooops typo. fixed.

Reply:
Wow, I missed it and commented for no reason :thinking: